test/fixtures/parser/snowflake/snowflake_secure_view.sql

```diff
@@ -0,0 +1,2 @@
+
+create secure view cool_view as select column_1, column_2 from some_table
```

Comment:
I think to be covering all of the examples that were given in https://github.com/sqlfluff/sqlfluff/issues/816 we'd just need one last test that is a `create secure view` with a `comment`